pkg/cmd/admin/policy/psp_review.go

```diff
@@ -0,0 +1,302 @@
+}
+
+func (o *reviewOptions) Run(out io.Writer) error {
+	review := &securityapi.PodSecurityPolicyReview{
+		Spec: securityapi.PodSecurityPolicyReviewSpec{
+			Template:            *o.PodTemplateSpec,
+			ServiceAccountNames: o.ServiceAccountNames,
+		},
+	}
+
+	response, err := o.client.PodSecurityPolicyReviews(o.Namespace).Create(review)
+	if err != nil {
+		return fmt.Errorf("unable to compute PSP rviews: %v", err)
+	}
+	return o.PrintObject(response, out)
+}
+
+func (o *reviewOptions) Complete(f *clientcmd.Factory, args []string, cmd *cobra.Command) error {
+	namespace, enforceNamespace, err := f.DefaultNamespace()
+	if err != nil {
+		return err
+	}
+	oclient, _, err := f.Clients()
+	if err != nil {
+		return fmt.Errorf("unable to obtain client: %v", err)
+	}
+	o.client = oclient
+	o.Namespace = namespace
+	mapper, typer := f.Object()
+
+	infos, err := resource.NewBuilder(mapper, typer, resource.ClientMapperFunc(f.ClientForMapping), kapi.Codecs.UniversalDecoder()).
+		NamespaceParam(namespace).
+		FilenameParam(enforceNamespace, &resource.FilenameOptions{Recursive: false, Filenames: []string{o.Filename}}).
+		SingleResourceType().
+		Do().
+		Infos()
+
+	if err != nil {
+		return fmt.Errorf("unable to find a resource: %v", err)
+	}
+	if len(infos) != 1 {
+		return fmt.Errorf("expected a resource for path: %s", o.Filename)
+	}
+	o.PodTemplateSpec, err = returnPodTemplateSpec(infos[0].Object)
+	if err != nil {
+		return fmt.Errorf("unable to find podTemplateSpec from %s: %v", o.Filename, err)
+	}
+
+	o.PrintObject = func(c *cobra.Command, m meta.RESTMapper) func(runtime.Object, io.Writer) error {
+		return func(obj runtime.Object, out io.Writer) error {
+			return f.PrintObject(c, m, obj, out)
+		}
+	}(cmd, mapper)
+	return nil
+}
+
+func NewCmdReview(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	o := &reviewOptions{}
+	cmd := &cobra.Command{
+		Use:     fmt.Sprintf("%s", name),
+		Short:   "Checks which ServiceAccount can create a Pod",
+		Long:    reviewLong,
+		Example: fmt.Sprintf(reviewExamples, fullName),
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := o.Complete(f, args, cmd); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+			err := o.Run(out)
+			kcmdutil.CheckErr(err)
+		},
+	}
+
+	cmd.Flags().StringVarP(&o.Filename, "filename", "f", o.Filename, "file containing the resource to be checked")
+	cmd.Flags().StringSliceVarP(&o.ServiceAccountNames, "serviceaccounts", "s", o.ServiceAccountNames, "List of ServiceAccount names, comma separated")
+	// TODO: defines a way in the API to defines all service accounts: removing omitempty for PodSecurityPolicyReviewSpec.ServiceAccountNames?
+	//cmd.Flags().BoolVar(&o.AllServiceAccounts, "allserviceaccounts", false, "If true, all existing service accounts will be considered")
+
+	kcmdutil.AddPrinterFlags(cmd)
+
+	return cmd
+}
+
+func returnPodTemplateSpec(obj runtime.Object) (*kapi.PodTemplateSpec, error) {
```

Comment:
It's in https://github.com/openshift/origin/blob/283a990246d914e2c2a5b00b75effedb8792b9b9/pkg/api/meta/pods.go#L52 actually.